crm/main.py

```diff
@@ -103,25 +104,33 @@ def show_acl_resources(self):
         print tabulate(data, headers=header, tablefmt="simple", missingval="")
         print '\n'
 
-    def show_acl_table_resources(self, table_id):
+    def show_acl_table_resources(self):
         """
         CRM Handler to display ACL table information.
         """
         countersdb = swsssdk.SonicV2Connector(host='127.0.0.1')
         countersdb.connect(countersdb.COUNTERS_DB)
 
-        header = ("Resource Name", "Used Count", "Available Count")
+        header = ("Table ID", "Resource Name", "Used Count", "Available Count")
```

Comment:
Just checking, is it possible to display Table name as well, if it's available in the DB?

Reply:
Looks like no, currently we cannot display ACL table name here.